src/main/index.js

```diff
@@ -14,21 +14,17 @@ function createMainWindow() {
     },
   })
 
-  if (isDevelopment) {
-    window.webContents.openDevTools()
-  }
+  isDevelopment && window.webContents.openDevTools();
 
-  if (isDevelopment) {
-    window.loadURL(`http://localhost:${process.env.ELECTRON_WEBPACK_WDS_PORT}`)
-  } else {
-    window.loadURL(
+  isDevelopment
```

Comment:
I think we can simplify this to:
```
 window.loadURL(isDevelopment ? localUrl : formatedUrl);
```

Reply:
and also add a simple comment

Reply:
Yes @btzr-io. Sounds correct I will make the changes.